src/standalone/Vegetation/canopy_parameterizations.jl

```diff
@@ -536,23 +571,21 @@ end
 
 Computes dark respiration (`Rd`),
 in units of mol CO2/m^2/s, as a function of the maximum rate of carboxylation of Rubisco (`Vcmax25`),
-and the moisture stress factor (`β`), an empirical factor `f` is equal to 0.015,
-a constant (`ΔHRd`), a standard temperature (`To`),
+An empirical factor `f` is equal to 0.015, a constant (`ΔHRd`), a standard temperature (`To`),
 the unversal gas constant (`R`), and the temperature (`T`).
 
 See Table 11.5 of G. Bonan's textbook, 
 Climate Change and Terrestrial Ecosystem Modeling (2019).
 """
 function dark_respiration(
     Vcmax25::FT,
-    β::FT,
     f::FT,
     ΔHRd::FT,
     T::FT,
     To::FT,
     R::FT,
 ) where {FT}
-    Rd = f * Vcmax25 * β * arrhenius_function(T, To, R, ΔHRd)
+    Rd = f * Vcmax25 * arrhenius_function(T, To, R, ΔHRd)
```

Comment:
So, removing β from An and using it in g1 only is ok, but dark respiration still needs to feel the impact of soil moisture. Include β in here again.